server.js

```diff
@@ -6217,6 +6217,45 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 
+// Buildkite integration.
+camp.route(/^\/buildkite\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var identifier = match[1];  // eg, 3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489
+  var branch = match[2];  // eg, master
```

Comment:
Is `branch` required for this badge to work, maybe we could make this optional?
If we do require a branch to be specified we could possibly use something like this to make it optional:
```javascript
camp.route(/^\/buildkite\/([^/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
  ...
  const branch = match[2] || 'master'; // Defaults to master if not specified
```

Reply:
If not required, no. I'll change it